MIGRATION.md

```diff
@@ -31,7 +31,7 @@ The following are required to build `python-coveralls`.
 
 ### virtualenv
 
-[`virtualenv`](http://www.virtualenv.org/en/latest/) is used to create
+[`virtualenv`](https://www.virtualenv.org/en/latest/) is used to create
```

Comment:
http://www.virtualenv.org/en/latest/ redirects to https://virtualenv.pypa.io/en/latest/

https://www.virtualenv.org/en/latest/ doesn't work

I think you can use https://virtualenv.pypa.io/en/latest/

Reply:
Done